website/docs/icons/library/index.js

```diff
@@ -127,24 +128,36 @@ export default class Index extends Component {
 
   @action
   selectIconSize(event) {
+    const { value: selectedIconSize } = event.target;
+
     this.router.transitionTo({
       queryParams: {
+        selectedIconSize,
         searchQuery: this.searchQuery,
         selectedGroupType: this.selectedGroupType,
-        selectedIconSize: event.target.value,
       },
     });
+
+    this.eventTracking.trackEvent(
+      `Icon Library - Size Selector - ${selectedIconSize}`
```

Comment:
@zamoore @Dhaulagiri should we add tracking also for the group selector?

Reply:
Yes. I think that was added after we had outlined the events to capture in the doc Zack was working from.